autodispose/src/main/java/com/uber/autodispose/ParallelFlowableScoper.java

```diff
@@ -36,7 +36,7 @@
         return;
       }
 
-      Subscriber<? super T>[] newSubscribers = new Subscriber[subscribers.length];
+      @SuppressWarnings("unchecked") Subscriber<? super T>[] newSubscribers = new Subscriber[subscribers.length];
```

Comment:
Can this be made to not have generics issues?

Reply:
I don't think so, because you can't create an array of a generic type T. There's always going to be some type munging behind the scenes. It's either cast here, or cast somewhere else and either way it's unchecked.